dangerfile.js

```diff
@@ -71,3 +74,21 @@ const snapshotsModified = fileChanges.some(filePath => (
 if (componentSourceFilesModified && !snapshotsModified) {
   warn('Package source files (e.g. `package/packageName/src/packageName.js`) were updated, but snapshots weren\'t. Have you checked that the tests still pass?'); // eslint-disable-line max-len
 }
+
+// Ensure shrinkwrap changes are intentional.
+const shrinkwrapUpdated = includes(modifiedFiles, 'changelog.md');
+if (shrinkwrapUpdated) {
+  warn('`npm-shrinkwrap.json` was updated. Ensure that this was intentional.');
+}
+
+// New files should include the Backpack license heading.
+const unlicensedFiles = createdFiles.filter((filePath) => {
+  if (filePath.match(/\.(js|css|scss|sh)/)) {
+    const fileContent = fs.readFileSync(filePath);
+    return !fileContent.includes('Backpack - Skyscanner\'s Design System');
```

Comment:
Hmm nitpick: i'd have chosen something a little more obvious such as `Licensed under the Apache License, Version 2.0 (the "License");`.